### PR TITLE
Jenkins

### PR DESCRIPTION
**Pull request by tneer**
Trying to clean up the Jenkins/docker stuff.  Still doesn't work.  Build at

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -6,29 +6,19 @@
...
 
-def workDir = "./"
 
 dockerizedBuildPipeline(
-  buildImageId: "${sonatypeDockerRegistryId()}/cdi/golang-1.14:1",
+  pathToDockerfile: 'jenkins/Dockerfile',
   prepare: {
     githubStatusUpdate('pending')
-    runSafely '''
-    curl -O https://releases.hashicorp.com/terraform/0.13.3/terraform_0.13.3_linux_amd64.zip
```
Comment (tneer) [inline Jenkinsfile:16]: The prepare step does not run in the docker container so this isn't doing you much good.

File: Jenkinsfile [hunk 2]
```diff
@@ -6,29 +6,19 @@
...
-    unzip terraform_0.13.3_linux_amd64.zip
-    '''
   },
-  dockerArgs: "-e WORKSPACE:workspace",
   buildAndTest: {
-    dir(workDir) {
```
Comment (tneer) [inline Jenkinsfile:22]: This is the equivalent of `cd .` In other words, it doesn't do anything.

File: Jenkinsfile [hunk 3]
```diff
@@ -6,29 +6,19 @@
...
-      runSafely '''
-      go get github.com/jstemmer/go-junit-report
-      go mod tidy
-      go mod vendor
-      go test ./... -v 2>&1 -p=1 | go-junit-report > test-results.xml
-      WORKSPACE=workspace make test
```
Comment (tneer) [inline Jenkinsfile]: `make test` is doing pretty much the same thing as `go test ./...` No need to run it twice.  Not a fan of Makefiles, there is enough customization in this one that it's probably worth figuring out.

File: Jenkinsfile [hunk 4]
```diff
@@ -45,7 +35,7 @@ dockerizedBuildPipeline(
   //     }
   //   }
   // },
-  testResults: [ 'test-results.xml' ],
```
Comment (tneer) [inline Jenkinsfile:47]: Just removed while trying to get `make test` to work without capturing junit results.

**Review by jamengual — APPROVED** [review]

**Pull request merged**